recipe/cc_wrap.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
-args="${@##-mmacosx-version-min=10.9*}"
-$NN_CC_ORIG $args -mmacosx-version-min=11.0
+args="${@##-mmacosx-version-min=13.3*}"
```

Comment:
I don't think the version in these 2 lines shouldn't match. My understanding of the purpose of the wrapper script is to remove the original value of `-mmacosx-version-min` and replace it with a new one. If the script is replacing the exact same string, we can probably remove the wrapper script all-together.

Reply:
I confirmed that the line `args="${@##-mmacosx-version-min=13.3*}"` doesn't do anything. That's why setting it to the same value that is being added makes no difference. I confirmed that the file can be simplified to:

```sh
#!/bin/sh

$NN_CC_ORIG "$@" -mmacosx-version-min=13.3
```